io.github.theforceengine.tfe.Mod.DarkForcesCommunityPatch.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "io.github.theforceengine.tfe.Mod.DarkForcesCommunityPatch",
+  "runtime": "io.github.theforceengine.tfe",
+  "sdk": "org.freedesktop.Sdk//23.08",
+  "build-options": {
+    "strip": false,
+    "no-debuginfo": true
+  },
+  "separate-locales": false,
+  "appstream-compose": false,
+  "build-extension": true,
+
+  "modules": [
+    {
+      "name": "DarkForcesCommunityPatch",
+      "sources": [
+        {
+          "type": "extra-data",
```

Comment:
you misunderstand how extra-data works. It's downloaded at install time so the build step below won't find it.